src/packages/shared-types/issue.ts

```diff
@@ -0,0 +1,83 @@
+import { z } from "zod";
+
+export const issueSchema = z.object({
+  id: z.string().uuid(),
+  title: z.string().min(1, { message: "Title is required" }),
+  description: z.string().min(1, { message: "Description is required" }),
+  type: z
+    .string()
+    .refine((val) => ["look", "functionality", "other"].includes(val), {
+      message: "Type must be one of \"look\", \"functionality\", or \"other\"",
+    }),
+  priority: z
+    .string()
+    .refine((val) => ["low", "medium", "high"].includes(val), {
+      message: "Priority must be one of \"low\", \"medium\", or \"high\"",
+    }),
+  resolved: z.boolean().default(false),
+  createdAt: z.string(),
+});
+
+export type Issue = z.infer<typeof issueSchema>;
+
+//////////////////////
+
+export const createIssueSchema = z.object({
+  title: z
+    .string()
+    .min(1, { message: "Title is required" })
+    .refine((value) => value !== "error", {
+      message: "Title cannot be \"error\"",
+    }),
+  description: z.string().min(1, { message: "Description is required" }),
+  type: z
+    .string()
+    .refine((val) => ["look", "functionality", "other"].includes(val), {
+      message: "Type must be one of \"look\", \"functionality\", or \"other\"",
+    }),
+  priority: z
+    .string()
+    .refine((val) => ["low", "medium", "high"].includes(val), {
+      message: "Priority must be one of \"low\", \"medium\", or \"high\"",
+    }),
+  resolved: z.boolean().default(false),
+});
+
+export type CreateIssueSchema = z.infer<typeof createIssueSchema>;
+export const validateCreateIssue = createIssueSchema.parse;
+
+////////////////////
+
+export const getIssueSchema = z.object({
+  id: z.string().min(1),
+  title: z.string().min(1, { message: "Title is required" }),
+  description: z.string().min(1, { message: "Description is required" }),
```

Comment:
Nice, it does error handling too? I can think of all sorts of places where this could be handy